scripts/deploy-with-s3.js

```diff
@@ -61,8 +60,6 @@ const cacheDirs = [
   ['.cache', '-cache/']
 ]
 
-const fs = require('fs')
```

Comment:
curious - how did you find that is not being used? linter?

Reply:
our `yarn ts-lint` complains about this once all actual errors are removed- at least for me.

Reply:
why didn't we detect it on CircleCI then? 🤔

Reply:
The `lint-ts` command is `tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx .`, so it must be `eslint` finding this after `tsc` checks TypeScript. I guess eslint warnings don't make the test fail on CI.

Reply:
good catch ... let's create a ticket to change this - it's p1-p0

Reply:
I'm on it!

Reply:
Done at #1291